test/smart-wallet.test.ts

```diff
@@ -0,0 +1,32 @@
+import { RelayingServices } from '../src';
+import { MockRelayingServices } from './mock';
+import Expect = jest.Expect;
+import {
+    MOCK_SMART_WALLET_ADDRESS,
+    MOCK_TOKEN_ADDRESS,
+} from './constants';
+
+declare const expect: Expect;
+
+describe('Tests for smart wallet', () => {
+    let sdk: RelayingServices;
+
+    beforeEach(async () => {
+        sdk = new MockRelayingServices();
+    });
+
+    it('Should fail with smart wallet already deployed', async () => {
+        try {
+            await sdk.deploySmartWallet(
+                {
+                    address: MOCK_SMART_WALLET_ADDRESS,
+                    index: 0
+                },
+                MOCK_TOKEN_ADDRESS,
+                0
+            );
```

Comment:
I would add some failing expectations here in case of a successful call of `sdk.deploySmartWallet`

Reply:
This will fixed in the next commit.